pkg/ruler/rulespb/rules.proto

```diff
@@ -30,6 +30,10 @@ message RuleGroupDesc {
   int64 limit =10;
   google.protobuf.Duration queryOffset = 11
       [(gogoproto.nullable) = true, (gogoproto.stdduration) = true];
```

Comment:
Unrelated, but interesting that queryOffset is using nullable = true while all the rest is not. I was reading through the implications https://pkg.go.dev/github.com/gogo/protobuf/gogoproto#hdr-More_Canonical_Go_Structures